Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -139,6 +149,10 @@ public final class AlertManager {
     // MARK: - Loop Not Running alerts
 
     func loopDidComplete() {
+        rescheduleLoopNotRunningNotifications()
+    }
+
+    func rescheduleLoopNotRunningNotifications() {
         clearLoopNotRunningNotifications()
         scheduleLoopNotRunningNotifications()
```

Comment:
I believe `scheduleLoopNotRunningNotifications` assumes it is running immediately after the loop completes. If this assumption is violated, and it is called at other times, you may need to pass in lastLoopCompleted, so it can calculate the correct alerting times.

Reply:
OK. I'll look and update.